### PR TITLE
Make it work under atom-shell

### PR DESCRIPTION
**Pull request by alfa07**

### CODE REVIEW

**Comment by portante** [comment]
@alfa07, can you resubmit this PR with no `build/*` files changed?

File: src/core.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+d3 = require('d3');
```
Comment (robinfhu) [inline src/core.js]: What happens if we include this in the browser normally and 'require' isn't defined? Won't this error out?

**Comment by robinfhu** [comment]
Please explain what this solves. Also, this breaks the tests.

File: src/core.js (hunk 1 above)
Comment (liquidpele) [inline src/core.js]: I assume they're actually tryign to use requirejs there?   There is another pull request for AMD, we should probably just make it loadable for requriejs and AMD so people don't have to use a shim.

**Pull request closed without merging**